components/filter/test/filter.axe.js

```diff
@@ -28,6 +28,17 @@ const multiDimensionFixture = html`
 	</d2l-filter>`;
 
 describe('d2l-filter', () => {
+	before(async() => {
+		await emulateMedia({ reducedMotion: 'reduce' });
```

Comment:
Would it make sense to try and do this is [in our helper](https://github.com/BrightspaceUI/visual-diff/blob/main/helpers/createPage.js#L3), or is it too late if we've already loaded the files?

Reply:
I'm confused by what you mean - you mean create a similar helper for axe tests?  Since they aren't puppeteer and don't have an `html` file, I was running into the issue where https://github.com/BrightspaceUI/core/blob/main/components/hierarchical-view/hierarchical-view-mixin.js#L6 was happening before the emulation was happening, and it's never measured again (which is fine outside of tests).  If we want all axe tests to have reduced motion on, I think we could add it here: https://modern-web.dev/docs/test-runner/browser-launchers/chrome/#customizing-browser-context-and-page

Reply:
Ah right sorry I got aXe and visual diff mixed up. But yeah the sentiment is the same -- to centralize this so that reduced motion is off for all aXe tests... if that's something we want. Probably added bonus would be a lot faster tests and a lot less waiting/timeouts.

Reply:
I can definitely look into it.  I'm not sure it'll actually help many other axe tests, it looks like we've avoided writing any that need to wait for something to animate/open - there's no tests for dropdown or hierarchical-view, or any where inputs are opened.  Which is probably fine if we have tests for the things inside those (menu, calendar, etc).  But filter just has a bit more going on so I wanted to get that working.  Applying it to all will help with anything similar in the future though

Reply:
Cool. I'm also good with merging this as-is and then if it comes up again and maybe again a 3rd time then we invest in making it more generic.